go/storage/serialcachewriter.go

```diff
@@ -10,33 +10,37 @@ import (
 
 const kSerials = "serials"
 
-type KnownCertificates struct {
+type SerialCacheWriter struct {
 	expDate   types.ExpDate
 	issuer    types.Issuer
 	cache     RemoteCache
 	expirySet bool
 }
 
-func NewKnownCertificates(aExpDate types.ExpDate, aIssuer types.Issuer, aCache RemoteCache) *KnownCertificates {
-	return &KnownCertificates{
+func NewSerialCacheWriter(aExpDate types.ExpDate, aIssuer types.Issuer, aCache RemoteCache) *SerialCacheWriter {
+	return &SerialCacheWriter{
 		expDate:   aExpDate,
 		issuer:    aIssuer,
 		cache:     aCache,
 		expirySet: false,
 	}
 }
 
-func (kc *KnownCertificates) id(params ...string) string {
+func NewSerialCacheReader(aExpDate types.ExpDate, aIssuer types.Issuer, aCache RemoteCache) *SerialCacheWriter {
```

Comment:
I do find it a bit odd that a function called `NewSerialCacheReader` returns a `SerialCacheWriter`, but I guess I understand the intention.

Reply:
The sole caller of this function is going to be removed in a follow-up commit. I did this as a note-to-self that the new code should have separate reader and writer types.